src/client/mod.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! The Client module to connect through HORNET or Bee with API usages
+
+mod high_level;
+
+use std::{
+    sync::{Arc, RwLock},
+    time::Duration,
+};
+
+use bee_block::{output::RentStructure, protocol::ProtocolParameters};
+use bee_pow::providers::{NonceProvider, NonceProviderBuilder};
+#[cfg(not(target_family = "wasm"))]
+use tokio::{
+    runtime::{Handle, Runtime},
+    sync::broadcast::Sender,
+};
+#[cfg(feature = "mqtt")]
+use {
+    crate::node_api::mqtt::{BrokerOptions, MqttEvent, TopicHandlerMap},
+    rumqttc::AsyncClient as MqttClient,
+    tokio::sync::watch::{Receiver as WatchReceiver, Sender as WatchSender},
+};
+
+use crate::{
+    builder::{ClientBuilder, NetworkInfo},
+    constants::DEFAULT_TIPS_INTERVAL,
+    error::Result,
+};
+
+/// An instance of the client using HORNET or Bee URI
+// #[cfg_attr(target_family = "wasm", derive(Clone))]
```

Comment:
Is that not needed?

Reply:
Yes not needed anymore, it was there back then when clone was only possible for wasm, but it's commented since 5 months